src/main/java/io/github/moehreag/axolotlclient/modules/hud/gui/hud/CPSHud.java

```diff
@@ -24,8 +23,6 @@ public class CPSHud extends CleanHudEntry {
     private final BooleanOption rmb = new BooleanOption("rightcps", false);
 
     boolean rc;
-    private long lastClicked;
-    private int pressedMouseButton;
```

Comment:
Technically, these are used, but the values that they are set to are never read, therefore setting these is useless.